<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentDataView</class>
 <widget class="QWidget" name="CurrentDataView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0">
      <widget class="QLabel" name="fuelEmissionLabel">
       <property name="text">
        <string>Расход топлива (кг/с)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="heightVal">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="accelVal">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="velocityVal">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Высота (м)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="velocityLabel">
       <property name="text">
        <string>Скорость (м/с)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="fuelMassVal">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QProgressBar" name="heightBar">
       <property name="styleSheet">
        <string notr="true"> QProgressBar::chunk {
	background-color: lightgreen;
	width: 1px;
 }

 QProgressBar {
	border: 1px solid grey;
	border-radius: 0px;
	text-align: center;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="fuelEmissionVal">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QProgressBar" name="fuelEmissionBar">
       <property name="styleSheet">
        <string notr="true"> QProgressBar::chunk {
	background-color: lightgreen;
	width: 1px;
 }

 QProgressBar {
	border: 1px solid grey;
	border-radius: 0px;
	text-align: center;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="fuelMassLabel">
       <property name="text">
        <string>Кол-во Топлива (кг)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="accelLabel">
       <property name="text">
        <string>Ускорение (м/с2)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QProgressBar" name="fuelMassBar">
       <property name="styleSheet">
        <string notr="true"> QProgressBar::chunk {
	background-color: lightgreen;
	width: 1px;
 }

 QProgressBar {
	border: 1px solid grey;
	border-radius: 0px;
	text-align: center;
 }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Время (сек)</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="timeVal">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
